src/main/java/at/hannibal2/skyhanni/features/mining/crystalhollows/CrystalHollowsLobbyAgeWarning.kt

```diff
@@ -0,0 +1,88 @@
+package at.hannibal2.skyhanni.features.mining.crystalhollows
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.IslandType
+import at.hannibal2.skyhanni.events.IslandChangeEvent
+import at.hannibal2.skyhanni.events.LorenzTickEvent
+import at.hannibal2.skyhanni.events.TabListUpdateEvent
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.LorenzUtils.isInIsland
+import at.hannibal2.skyhanni.utils.NumberUtil.formatNumber
+import at.hannibal2.skyhanni.utils.StringUtils.matchMatcher
+import at.hannibal2.skyhanni.utils.TabListData
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import net.minecraft.client.Minecraft
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import kotlin.math.abs
+
+class CrystalHollowsLobbyAgeWarning {
+    private val mc get() = Minecraft.getMinecraft()
+
+    private val THE_CRYSTAL_HOLLOWS = IslandType.CRYSTAL_HOLLOWS
+
+    private val TICKS_PER_MC_DAY_MINECRAFT: Long = 24000L
+
+    private val crystalHollowsLobbyRepoGroup = RepoPattern.group("crystalhollows.lobbyage")
+    private val playerCountTabListPattern by crystalHollowsLobbyRepoGroup.pattern(("playercount.tablist"), (" *(?:§.)*Players[\\S ]*[\\(\\[\\{](?<playerCount>[\\S ]+)[\\)\\]\\}]"))
```

Comment:
not sure if we need the group here as its only 1 pattern
also there seems to be extra sets of brackets in the .pattern
also up to you but if you format these lines more like i have in https://github.com/hannibal002/SkyHanni/pull/871 (naming of group if you still want it and formatting of the pattern line itself) that would be cool